src/overlays/actors/ovl_En_Sw/z_en_sw.c

```diff
@@ -433,6 +431,8 @@ void func_808D94D0(EnSw* this, PlayState* play, s32 arg2, s32 arg3, s16 arg4) {
                 break;
             }
         }
+
+        if (i == 3) {}
```

Comment:
lol, does C even keep the reference to variables out of scope like this?

I know you want to document in the next build but can we add a tag to this for now?
```suggestion
        //! FAKE
        if (i == 3) {}
```

Reply:
I was very surpised that it worked, but the Bomchu code gave me the hint... In that instance, if ever it satisfies this condition, the bomchu explodes. I'm not sure how picky to get over the usage of "FAKE", but I'd assume this is here for something like a debug assert/message? If that still tracks with what we wanto to use FAKE for, gimme a thumbs up and I'll do that.

Reply:
No its just a comment for us to search for later, it means we think its fake and there is a better match to be found later, a bunch of recent `if (1) {}` fake matches were recently discovered to be early returns by Engineer124, but this looks different to me, and we might figure it out later, this comment just helps us find it easier by searching a large project.